store/sqlstore/team_store.go

```diff
@@ -1325,21 +1422,15 @@ func (s SqlTeamStore) UserBelongsToTeams(userId string, teamIds []string) (bool,
 }
 
 func (s SqlTeamStore) UpdateMembersRole(teamID string, userIDs []string) *model.AppError {
-	sql := fmt.Sprintf(`
-		UPDATE
-			TeamMembers
-		SET
-			SchemeAdmin = CASE WHEN UserId IN ('%s') THEN
-				TRUE
-			ELSE
-				FALSE
-			END
-		WHERE
-			TeamId = :TeamId
-			AND (SchemeGuest = false OR SchemeGuest IS NULL)
-			AND DeleteAt = 0`, strings.Join(userIDs, "', '"))
-
-	if _, err := s.GetMaster().Exec(sql, map[string]interface{}{"TeamId": teamID}); err != nil {
+	sql, args, err := s.getQueryBuilder().
+		Update("TeamMembers").
+		Set("SchemeAdmin", sq.Case().When(fmt.Sprintf("UserId IN ('%s')", strings.Join(userIDs, "', '")), "true").Else("false")).
```

Comment:
I think squirrel is able to build the `in` clauses just using the `sq.Eq`:

```suggestion
		Set("SchemeAdmin", sq.Case().When(sq.Eq{"UserId": userIDs}, "true").Else("false")).
```

I think that suggestion should work

Reply:
Nice to know this! Thanks!